template/tabs/tab.html

```diff
@@ -1,3 +1,3 @@
 <li ng-class="{active: active, disabled: disabled}">
-  <a href ng-click="select()" tab-heading-transclude>{{heading}}</a>
+  <a href ng-click="select($event)" tab-heading-transclude>{{heading}}</a>
```

Comment:
With this change, documentation should be added for the `select` option as well.